ui/src/pages/plan/rules/IRule.ts

```diff
@@ -10,5 +10,4 @@ export interface IApiRuleMutate {
 export interface IApiRule extends IApiRuleMutate {
     id: string;
     userid: string;
-    version: string;
```

Comment:
I figured migrations take care of this, so I took it out. Can be added later if we really need it.